src/SchemaDerivedConfig.js

```diff
@@ -56,7 +57,10 @@ class SchemaDerivedConfig extends BaseConfig {
       verbose: true,
       useDefaults: true,
       coerceTypes: 'array',
+      allowUnionTypes: true,
     });
+    ajv.addVocabulary(['meta-license', 'meta-status', 'meta-enum']);
```

Comment:
I guess having custom vocabularies with a colon separator is not supported. If that's the case, I'd be hesitant to apply the update as it would break compatibility with XDM and jsonschema2md.

Reply:
yes. per json meta schema, `:` is not allowed (or something like this :-).
we can just disable strict mode, and then everything should still work.

https://github.com/ajv-validator/ajv/blob/master/docs/api.md#ajvaddkeyworddefinition-objects-ajv

Reply:
Or we can use a new metaschema...maybe

Reply:
I just disabled strict mode....